src/AutoRest.CSharp/Common/Output/Models/MethodSignatureBase.cs

```diff
@@ -25,6 +25,8 @@ internal enum MethodSignatureModifiers
         Static = 16,
         Extension = 32,
         Virtual = 64,
-        Async = 128
+        Async = 128,
+        New = 256,
+        Override = 512
```

Comment:
My PR of the polymorphic resource will introduce another value of `abstract` here. FYI

Reply:
was thinking of doing it that way for this comment as well https://github.com/Azure/autorest.csharp/pull/2773#discussion_r995471586